Source/WebCore/Modules/webxr/WebXRSession.h

```diff
@@ -167,7 +168,7 @@ class WebXRSession final : public RefCounted<WebXRSession>, public EventTarget,
     std::optional<PlatformXR::RequestData> m_requestData;
 
     double m_minimumInlineFOV { 0.0 };
-    double m_maximumInlineFOV { piFloat };
+    double m_maximumInlineFOV { std::numbers::pi_v<float> };
```

Comment:
This is also existing, but using the float version for this double value seems incorrect.